vizro-core/src/vizro/__init__.py

```diff
@@ -1,20 +1,36 @@
 import logging
 import os
 
-from dash.development.base_component import ComponentRegistry
+from dash.development.base_component import Component
 
 from ._vizro import Vizro
 
-ComponentRegistry.registry.add("vizro")
+logging.basicConfig(level=os.getenv("VIZRO_LOG_LEVEL", "WARNING"))
+
+__all__ = ["Vizro"]
+
+__version__ = "0.1.20.dev0"
```

Comment:
@huong-li-nguyen You'll need to make the tag `0.1.12.dev0` to test this before releasing with `serve_locally=False`.

Reply:
Actually tagging is a bad way to do this because it points to a fixed commit... Let's try [4c260c1](https://github.com/mckinsey/vizro/pull/578/commits/4c260c16c38d0ce30f46e0ca00654ba1987dc9fa).